src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -0,0 +1,101 @@
+@import '~amo/css/styles';
+
+.GuidesAddonCard {
+  display: flex;
+  flex-direction: column;
+
+  // stylelint-disable max-line-length
+  // TODO: check on get-firefox button style.
+  // See https://github.com/mozilla/addons-frontend/issues/6432#issuecomment-433083079.
+  .Button--get-firefox,
+  .AMInstallButton {
+    @include respond-to(medium) {
+      align-self: center;
+      margin-left: auto;
+    }
+  }
+
+  .Icon-trophy {
+    height: 12px;
+    margin-right: 4px;
+    width: 12px;
+  }
+
+  .AddonTitle {
+    font-size: 16px;
+    font-weight: 500;
+    margin-bottom: 0;
+  }
+
+  .GuidesAddonCard-content-header-title {
+    .AddonTitle-author {
+      color: $grey-50;
+      font-size: 14px;
+      padding-top: 0;
+
+      a {
+        color: $blue-50;
+        font-size: inherit;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  .Notice-error {
+    margin-bottom: 12px;
+  }
+}
+
+.GuidesAddonCard-content {
+  display: flex;
+  flex-wrap: wrap;
+
+  @include respond-to(medium) {
+    flex-wrap: nowrap;
+  }
+}
+
+.GuidesAddonCard-content-icon {
+  height: 40px;
+  margin-bottom: 12px;
+  width: 40px;
+}
+
+.GuidesAddonCard-content-header {
+  @include respond-to(medium) {
+    display: flex;
+  }
+}
+
+.GuidesAddonCard-content-header-title {
+  display: flex;
+  flex-direction: column;
+
+  @include respond-to(medium) {
+    margin: 0 16px;
+  }
+}
+
+.GuidesAddonCard-content-header-staff-pick {
+  color: $grey-50;
+  font-size: 10px;
+  font-weight: 500;
+  margin-top: 8px;
+  text-transform: uppercase;
+  white-space: nowrap;
+
+  @include respond-to(medium) {
+    margin-top: 2px;
+  }
+}
+
+.GuidesAddonCard-content-description {
+  color: $grey-60;
+  display: flex;
+  font-size: 14px;
+  line-height: 1.4;
```

Comment:
We do line-height in pixels -- does the `$default-line-height` variable work? If not, you can set a custom one in pixels.

Reply:
actually, I just saw some discussion about this topic on irc..should this still be switched or is this okay?

Reply:
Yes, I learned that a large chunk of the app uses this approach so I suppose it's fine. This number seems to match some other related values.